crates/outbound-http/src/lib.rs

```diff
@@ -53,6 +53,9 @@ impl outbound_http::Host for OutboundHttp {
                 .map_err(|_| HttpError::RuntimeError)?
             {
                 tracing::log::info!("Destination not allowed: {}", req.uri);
+                terminal::warn!("A component tried to make a HTTP request to '{}'.", req.uri);
+                eprintln!("The component was configured not to allow requests to this host.");
+                eprintln!("To allow requests, add the host to `allowed_http_posts`.");
```

Comment:
```suggestion
                eprintln!("To allow requests, add the host name to the `allowed_http_posts` key/value pair in the component section of spin.toml.");
```

Reply:
I did wonder to what extent to make this a "explain what is happening" vs "write exhaustive documentation on how to fix it."  E.g. I thought about warning them to restart after making the change, but was wary of turning it into a wall of text that the eyes just slid off.  It may be too much in that direction already; I'd have preferred to keep it to 1 or 2 lines.